### DISCUSSION TITLE
[W3MService] Please provide a valid projectId. See https://docs.walletconnect.com/web3modal/flutter/options for details.

### DISCUSSION THREAD

**Discussion by sureshturbo** [WalletConnect/walletconnect-monorepo#3898] [Developer Support]
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/44191960/7b629e77-6e81-4ce1-ad57-4f79a1e474e9)

 flutter run --dart-define=PROJECT_ID='657cca24a6aebfa977eb5b427b87cXXX'
 
 i put project id but it shows this error in flutter
 plz help me.

**Comment by glitch-txs** [comment]
Hi, please open an issue in the flutter repository
